sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/MessageUtils.java

```diff
@@ -326,4 +331,35 @@ public static ServiceBusMessage traceMessageSpan(ServiceBusMessage serviceBusMes
         }
         return serviceBusMessage;
     }
+
+    /**
+     * Convert described type to URI object.
+     * @param described the described of described type.
+     * @return URI or if create failed return String.
+     */
+    public static URI describedToURI(Object described) {
+        return URI.create((String) described);
```

Comment:
error handling?

Reply:
the error handling is in the code where use these functions.